readwriter.h

```diff
@@ -3,6 +3,17 @@
 
 #include "testlib.h"
 #include "bits/stdc++.h"
+#include <filesystem>
+
+enum PrintFormat {
+    Prompt,
+    Solution
+};
+
+const std::map<std::string, std::string> dirs = {
```

Comment:
What is this map for?

Reply:
Added more descriptive values. Added Comment